src/chess/chess-arbiter.js

```diff
@@ -27,28 +32,57 @@ AFRAME.registerSystem("chess-arbiter", {
     }
   },
 
-  startGame() {
-		this.chessEngine = new Chess();
-	},
+  addEventListeners() {
+    this.el.sceneEl.addEventListener("chess:playAs", this.playAsEvent);
+    this.el.sceneEl.addEventListener("chess:copyPGN", this.copyPGN);
+    this.el.sceneEl.addEventListener("chess:copyFEN", this.copyFEN);
+  },
+
+  removeEventListeners() {
+    this.el.sceneEl.removeEventListener("chess:playAs", this.playAsEvent);
+  },
```

Comment:
Should the add & remove methods do all the same listeners?

Reply:
Yeah, oversight on my part. I'll correct it and update the PR.

Reply:
Fixed in commit 3a62277.